### DISCUSSION TITLE
STANDALONE AIRGAP INSTALLATION stuck in a loop , I receive the same issue with Distributed Airgap option too

### DISCUSSION THREAD

**Discussion by kenp12** [Security-Onion-Solutions/securityonion#11144] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

![Screenshot 2023-08-22 233210](https://github.com/Security-Onion-Solutions/securityonion/assets/132363283/c9206b51-f690-4e7d-82bd-89199840f1b9)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment]
Are you positive you selected airgap?

**Reply by kenp12** [reply]
Yes, I am very positive that I selected airgap on all 5 attempts. I get the same issue.

**Reply by TOoSmOotH** [reply]
I am unable to reproduce the issue. Does your md5 of the iso match that of the one listed in the release notes?

**Comment by kenp12** [comment, marked as answer]
I had a bad iso thanks for the questions
